pages/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -1313,11 +1313,11 @@ msgid "Shaun Zacharia, President & Co-Founder, Triplelift."
 msgstr ""
 
 #: pages/content/amp-dev/about/ads.html:208
-msgid "The average mobile site takes 19 seconds to load over a 3G connection. Coupled with the fact that conversions fall by 12% for every additional second a webpage takes to load, you can safely assume two things  speed equals performance, and most advertiser campaigns have room to improve. That’s where you come in. As an SSP, DSP, analytics provider, or anything in between, embracing speed means improving campaign performance, delivering quality user experiences, and establishing yourself as the adtech platform for the fast web. Join the hundreds of technology platforms currently supporting the open source AMP ecosystem. "
+msgid "The average mobile site takes 19 seconds to load over a 3G connection. Coupled with the fact that conversions fall by 12% for every additional second a webpage takes to load, you can safely assume two things:  speed equals performance, and most advertiser campaigns have room to improve. That’s where you come in. As an SSP, DSP, analytics provider, or anything in between, embracing speed means improving campaign performance, delivering quality user experiences, and establishing yourself as the adtech platform for the fast web. Join the hundreds of technology platforms currently supporting the open source AMP ecosystem. "
```

Comment:
Looks like theres two spaces here `things:  speed`

Reply:
Also a space here: `ecosystem. "`

Reply:
Ah, good catch. That is actually how it is in the html itself. I'll update all locations